eZ/Publish/API/Repository/Values/ContentType/ContentType.php

```diff
@@ -179,13 +138,6 @@
     protected $isContainer;
 
     /**
-     * Main language.
-     *
-     * @var string
-     */
-    protected $mainLanguageCode;
```

Comment:
@andrerom  classes extending API ContentType redefine it via trait. 
In PHP 5.6 redefining properties [triggers `E_STRICT` error](https://travis-ci.org/ezsystems/ezpublish-kernel/jobs/237973738#L669).